### PR TITLE
On preparation for release 0.4.0

### PR DESCRIPTION
**Pull request by armfazh**
On preparation for release 0.4.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,29 +31,29 @@
         "build": "tsc -b",
         "test": "tsc -b test && node --experimental-vm-modules node_modules/jest/bin/jest.js --ci",
         "lint": "eslint .",
-        "format": "eslint src/**/*.ts test/**/*.ts examples/**/*.ts --fix",
+        "format": "prettier './(src|test|examples)/**/!(*.d).ts' --write",
```
Comment (thibmeu) [inline package.json:34]: Let's stick to the same command for formatting and linting

**Review by thibmeu — APPROVED** [review]
Looks good overall, one nit

File: package.json (hunk 1 above)
Comment (armfazh) [reply package.json:34]: I tried the eslint version and didn't work for one file I edit on purpose, but prettier was able to format it.

Comment (armfazh) [reply package.json:34]: let me know if there is a simpler configuration.

**Pull request merged**